src/runtime_src/core/tools/common/tests/TestAIEReconfigOverhead.cpp

```diff
@@ -0,0 +1,170 @@
+// ------ I N C L U D E   F I L E S -------------------------------------------
+// Local - Include Files
+
+#include "TestAIEReconfigOverhead.h"
+#include "tools/common/XBUtilities.h"
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_hw_context.h"
+#include "xrt/xrt_kernel.h"
+
+// System - Include Files
+#include <fstream>
+#include <filesystem>
+#include <thread>
+#include <iostream>
+static constexpr size_t buffer_size_gb = 1;
+static constexpr size_t buffer_size = buffer_size_gb * 1024 * 1024 * 1024; //1 GB
+static constexpr size_t word_count = buffer_size/4;
+static constexpr int itr_count = 500;
+
+TestAIEReconfigOverhead::TestAIEReconfigOverhead()
+  : TestRunner("aie-reconfig-overhead", "Run end-to-end array reconfiguration overhead through shim DMA")
+{}
+
+boost::property_tree::ptree
+TestAIEReconfigOverhead::run(std::shared_ptr<xrt_core::device> dev)
+{
+  boost::property_tree::ptree ptree = get_test_header();
```

Comment:
Please do ptree.erase("xclbin") after get_test_header() as part of our hotfix to remove the empty xclbin field from the json; you can verify this by checking the json output! We'll fix this with a better approach soon.